shell/ci/release/docker.sh

```diff
@@ -171,6 +165,26 @@ build_and_push_image() {
         "$buildContext"
     )
   fi
+
+  # Tag the images for devenv
+  # For primary container, the image should be built with gcr.io/outreach-docker/$APPNAME:latest
+  # For dependency container, the image should be built with gcr.io/outreach-docker/$APPNAME/$image:local 
+  if [[ -n $DEVENV_DOCKER_BUILD ]]; then
+    info "Tagging the images for devenv"
+    if [[ "$image" != "$APPNAME" ]]; then
+      devenv_image="gcr.io/outreach-docker/$APPNAME/$image"
+      (
+        set -x
+        docker tag "$image":latest "$devenv_image":local
```

Comment:
Does this work? I believe that the `kind` loading logic in the devenv would also need to be changed?

Reply:
I run `devenv apps deploy .`, which seems to work. let me clear my kind cache and test again.

Reply:
I ran it again on a newly provisioned devenv. The images were loaded on nodes. Got some unhealthy checks, but don't think it is related. hmm, now I wonder how the images were loaded
```
Events:
  Type     Reason     Age                    From               Message
  ----     ------     ----                   ----               -------
  Normal   Scheduled  5m25s                  default-scheduler  Successfully assigned viamplify--bento1a/viamplify-d8cb96bc5-nmq9v to dev-environment-control-plane
  Normal   Pulled     5m24s                  kubelet            Container image "gcr.io/outreach-docker/viamplify:local" already present on machine
  Normal   Created    5m24s                  kubelet            Created container viamplify
  Normal   Created    5m23s                  kubelet            Created container duckling
  Normal   Created    5m23s                  kubelet            Created container co-service
  Normal   Started    5m23s                  kubelet            Started container co-service
  Normal   Pulled     5m23s                  kubelet            Container image "gcr.io/outreach-docker/duckling:20210318T214905-0bf91a411f2a1f1f808b98bc0dc579b61cdecaec" already present on machine
  Normal   Started    5m23s                  kubelet            Started container viamplify
  Normal   Pulled     5m23s                  kubelet            Container image "gcr.io/outreach-docker/viamplify/co-service:local" already present on machine
  Normal   Started    5m22s                  kubelet            Started container duckling
  Warning  Unhealthy  5m9s                   kubelet            Readiness probe failed: Get "http://10.244.0.158:9000/ping": dial tcp 10.244.0.158:9000: connect: connection refused
  ```

Reply:
I guess devenv destroy doesn't cleanup images. I need to exec into control panel and manually remove the image.  I will address the kind loading part.